lib/config/defaults.js

```diff
@@ -326,9 +331,16 @@ const applyOutputDefaults = (output, { target, outputModule, development }) => {
 		return library || "";
 	};
 
-	const getLibraryIdentifier = library => {
-		return Template.toIdentifier(getLibraryName(library));
-	};
+	F(output, "uniqueName", () => {
+		const libraryName = getLibraryName(output.library);
+		if (libraryName) return libraryName;
+		try {
+			const packageInfo = require(`${context}/package.json`);
+			return packageInfo.name || "";
```

Comment:
Name can be `@org/package`, the `Template.toIdentifier` method remove invalid characters?

Reply:
yep it does remove/replace invalidate chars.